### PR TITLE
chromatic: added chromatic workflow

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
This PR adds a github action that publishes our storybook to Chromatic on each push on the **develop** branch.

**Solving issue**  
Fixes: [CDEP-2659](https://tegel.atlassian.net/browse/CDEP-2659)

[CDEP-2659]: https://tegel.atlassian.net/browse/CDEP-2659?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v1
+        - name: Install dependencies
+          run: npm install
+        - name: Installing dependencies in core
+          working-directory: packages/core 
+          run: npm install
+        - name: Building @scania/tegl
```
Comment (timrombergjakobsson) [inline .github/workflows/chromatic.yml:14]:
```suggestion
        - name: Building @scania/tegel
```

**Comment by timrombergjakobsson** [comment]
So this will be triggered on each push in a branch? Or when is it triggered?

**Comment by mJarsater** [comment]
> So this will be triggered on each push in a branch? Or when is it triggered?

Yes, we'll demo it an a checkin also.

**Review by alexandereneroth — APPROVED** [review]
LGTM lets try this out

**Pull request merged**